prime-angular-frontend/src/app/modules/phsa-labtech/pages/bcsc-demographic/bcsc-demographic.component.html

```diff
@@ -0,0 +1,46 @@
+<app-page [busy]="busy"
+          [mode]="'default'"> 
+    <app-page-header>PRIME Enrolment</app-page-header>
+    <app-progress-indicator [noContent]="true"></app-progress-indicator>
+  
+    <form (ngSubmit)="onSubmit()"
+          [formGroup]="form"
+          novalidate>
+  
+      <section class="mb-3">
+  
+        <app-page-subheader>
+          <ng-container appPageSubheaderTitle>Enrollee Information</ng-container>
+          <ng-container appPageSubheaderSummary>
+            This information comes from your BC Services Card. 
+          </ng-container>
+        </app-page-subheader>
+  
+        <app-enrollee-profile [enrollee]="enrollee"
+                              [showAddress]="false"></app-enrollee-profile>  
+      </section>
+  
+      <section class="mb-3">
+  
+        <app-page-subheader>
+          <ng-container appPageSubheaderTitle>Address</ng-container>
+          <ng-container appPageSubheaderSummary>
+            This information comes from the your BC Services Card. 
+          </ng-container>
+        </app-page-subheader>
+  
+        <app-enrollee-profile [enrollee]="enrollee"
+                              [showPersonal]="false"
+                              [showAddressTitle]="false"></app-enrollee-profile>  
+      </section>
+  
+      <app-contact-information-form [form]="form"
+                                    [showSmsPhone]="false"></app-contact-information-form>
+  
+    </form>
+  
+    <app-page-footer [primaryActionLabel]="'Submit'"
+                     [hasSecondaryAction]="false"
+                     (save)="onSubmit()"></app-page-footer>
+</app-page> 
+  
```

Comment:
Applied formatting with right-click > Format Document ... how can I enable automatic formatting?  Didn't find anything with quick Google search

Reply:
BTW, I realized that because I was opening code base at root of repo rather than /moh-prime/prime-angular-frontend, I wasn't getting the Extension recommendations.

Reply:
In VSCode there's a workspace setting called "Format On Save" that you can turn on